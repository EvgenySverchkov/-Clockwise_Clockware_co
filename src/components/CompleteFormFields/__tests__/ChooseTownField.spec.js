import React from "react";

import {create} from "react-test-renderer";
import ChooseTownsField from "../ChooseTownField";

it("<ChooseTownsField /> test which is used in forms", ()=>{
    const mockProps = {
        changeHandler: jest.fn(), 
        isLoad: false, 
        townsArr: [{id: 1, name: "SomeTown"}, {id: 2, name: "SomeTown2"}]
    }
    const component = create(<ChooseTownsField {...mockProps}/>);
    expect(component.toJSON()).toMatchSnapshot();
})