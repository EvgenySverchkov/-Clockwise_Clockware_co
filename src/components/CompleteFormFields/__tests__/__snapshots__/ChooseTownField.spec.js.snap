// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChooseTownsField /> test which is used in forms 1`] = `
<div
  className="form-group  text-center text-sm-left col-12 col-md-10 col-lg-8"
>
  <div
    className="mb-2 font-weight-bold"
  >
    Choose town
  </div>
  <div
    className="form-check-inline"
  >
    <div
      className="form-check form-check-inline"
    >
      <input
        className="form-check-input"
        id="SomeTown"
        name="town"
        onChange={[MockFunction]}
        type="radio"
        value="SomeTown"
      />
      <label
        className="pl-0 pr-0 col-sm-4 col-md-4 col-form-label "
        htmlFor="SomeTown"
      >
        SomeTown
      </label>
    </div>
  </div>
  <div
    className="form-check-inline"
  >
    <div
      className="form-check form-check-inline"
    >
      <input
        className="form-check-input"
        id="SomeTown2"
        name="town"
        onChange={[MockFunction]}
        type="radio"
        value="SomeTown2"
      />
      <label
        className="pl-0 pr-0 col-sm-4 col-md-4 col-form-label "
        htmlFor="SomeTown2"
      >
        SomeTown2
      </label>
    </div>
  </div>
</div>
`;
